cmd/object-handlers.go

```diff
@@ -1226,6 +1250,14 @@ func (api objectAPIHandlers) CompleteMultipartUploadHandler(w http.ResponseWrite
 		return
 	}
 
+	// Deny if WORM is enbled
```

Comment:
s/enbled/enabled - needs to be replaced everywhere.

Reply:
Done